sdk/python/kfp/components/pipeline_task_test.py

```diff
@@ -411,6 +411,78 @@ def my_pipeline():
                 compiler.Compiler().compile(
                     pipeline_func=my_pipeline, package_path=package_path)
 
+    def test_outside_of_for_loop_blocked(self):
+        with self.assertRaisesRegex(ValueError,
+                                    r'Cannot use \.after\(\) across'):
+
+            @dsl.component
+            def print_op(message: str):
+                print(message)
+
+            @dsl.component
+            def return_list() -> int:
+                return [1, 2, 3]
+
+            @dsl.pipeline(name='pipeline-with-multiple-exit-handlers')
```

Comment:
nit: update names throughout or remove (`dsl.pipeline` does not need to be called)

Reply:
Probably should remove for all in this file -- I see that the other test cases have misleading names also